src/components/try-hcp-callout/components/a-b-test.tsx

```diff
@@ -0,0 +1,29 @@
+import { useEffect } from 'react'
+import { isInUS } from '@hashicorp/platform-util/geo'
+import { useFlags } from 'flags/client'
+import { abTestTrack } from 'lib/ab-test-track'
+
+const ctaSuffix = 'for free'
+
+export function useABTestCta(ctaText: string): string {
+	const { flags, settled } = useFlags()
```

Comment:
I think this hook might be intended to be only called once per page. Maybe not an issue now, but if we add more tests we'll need to lift this up to _app or a page component and pass the flag data down.

Reply:
🤔 good point, i forgot about that! I'll check to see if other flags are in use now and reconsider if so.

Reply:
I made a PR to[ remove the `enableAuth` check here,](https://github.com/hashicorp/dev-portal/pull/1607) which was the only other call for `useFlags` that I saw. Also I added a comment with your suggestion as a future note. Since this is the only test running atm, I'm okay with not lifting it up.